src/Nancy/RequestHeaders.cs

```diff
@@ -372,17 +428,19 @@ private static IEnumerable<INancyCookie> GetNancyCookies(IEnumerable<string> coo
 
         private IEnumerable<string> GetValue(string name)
         {
-            return this.GetValue(name, x => x);
+            return this.GetValue(name, x => x, new string[] {});

```

Comment:
Yeah that's `4.6+`. I initially had it as `Enumerable.Empty<string>()` and that performed worse, so I decided to just K.I.S.S and stick with the primitive